test/rubygems/test_require.rb

```diff
@@ -509,6 +509,33 @@ def test_default_gem_and_normal_gem
     assert_equal %w[default-3.0], loaded_spec_names
   end
 
+  def test_normal_gems_with_overridden_load_error_message
+    normal_gem_spec = util_spec("default", "3.0", nil, "lib/default/gem.rb")
```

Comment:
Could you use names and paths different than "default" just to make it clear that default gems are not involved in this test? The "normal gem_spec" name for the variable already emphasizes this, but still.

Reply:
```suggestion
    normal_gem_spec = util_spec("normal", "3.0", nil, "lib/normal/gem.rb")
```

Reply:
All done :+1: